merlin/study/study.py

```diff
@@ -418,7 +418,19 @@ def generate_samples(self):
                     "cmd"
                 ]
                 LOG.info("Generating samples...")
-                subprocess.call(sample_generate, shell=True)
+                sample_process = subprocess.Popen(
+                    sample_generate,
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.PIPE,
+                    shell=True,
```

Comment:
Ben, have you tested this in one of the parallel workflows?

Reply:
I hadn't, looking at this now.

Reply:
Everything in `slurm_par` appears to work with this branch.

Reply:
Thanks for checking.